src/containers/Messages/Messages.js

```diff
@@ -68,15 +79,28 @@ const Messages = () => {
     firebase.auth().onAuthStateChanged(async function(user) {
       if(!user) return window.location.href = '/login';
 
-      const admin = await isAdmin(firebase);
+      const admin = await checkIsAdmin(firebase);
       const roomOwner = user.uid === room;
       if(!admin && !roomOwner) return window.location.href = '/login';
 
+      setIsAdmin(admin);
       getUser(user);
       setUser(user);
-      getMessages(room, setConversation);
+      getMessages(room, setConversation, setAdminId);
+
+      messaging.requestPermission()
+        .then(async function() {
+          const token = await messaging.getToken();
+          addUserToken({ uid: user.uid, token });
+        })
+        .catch(function (err) {
+          // TODO: Implment toaster
```

Comment:
Please, what is that toaster?

Reply:
toaster is the notification popup shows up when we want to show some success or error message to user.